lib/restbase.js

```diff
@@ -307,6 +318,18 @@ RESTBase.prototype._request = function (req) {
             self.metrics.endTiming([statName + statusClass, statName + 'ALL'], startTime);
             throw err;
         });
+        if (match.permissions
+            && Array.isArray(match.permissions)
+            && match.permissions.length > 0) {
+            request = request.then(function(res) {
+                var permissionCheckRestbase = self.makeChild(req);
```

Comment:
@d00rman Indeed you are right, I thought it would be bad to use 1 restbase for 2 requests (handler and permission check), but looks like it's fine..